pkg/handlers/clusterlogging/handler.go

```diff
@@ -37,38 +40,66 @@ func NewHandlers(opts *config.Options) []handlers.RequestHandler {
 }
 
 func (ext *handler) Process(req *http.Request, context *handlers.RequestContext) (*http.Request, error) {
-	name := context.UserName
-	if context.IsWhiteListed(name) || ext.hasInfraRole(context) {
-		log.Debugf("Skipping additional processing, %s is whitelisted or has the infra role", name)
-		return req, nil
+
+	log.Infof("Received context: UserName: %v", context.UserName)
```

Comment:
Should these be debug?

Reply:
that's just for me, it'll be removed for final review